### ISSUE TITLE
Organizational: Due dates for other HWs

### ISSUE THREAD

**Issue by dlee138** [openconnectome/Statistical-Connectomics-Class#215]
Some of the HWs in the past haven't had a specified due date other than the fact that they should be turned by the end of the semester.  Are these HWs due on the same day as the final project (May 14th)?

**Comment by gkiar** [comment]
Yep. I've been adding due dates to them since we read the feedback and
people wanted more specific deliverables, so moving forward they do have
dates. Otherwise, if not previously specified, the end of semester (i.e.
May 14th 11:59:59 pm) is fine. Thanks!!
G.

--
Greg Kiar <gkiar07@gmail.com>

**Comment by whock** [comment]
To piggyback on this question, for the github questions do you know how we can see how many questions we've answered? I can see how many I've opened by searching my name but not how many I've answered. I'm new to github so it might be a really simple thing.

**Comment by gkiar** [comment]
Put commenter:username in the search field instead of author:username to
see the responses you've given. :)

--
Greg Kiar <gkiar07@gmail.com>
